orbital-mechanics/Lecture10/Lecture 10 Spacecraft Dynamics.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>Looks good but the same comments from Lecture 1 apply here for formatting in general and for equations, they are not appearing numbered. Also the same thing with html rendering.</p><p>have you tried what Joost sent? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>